src/containers/auth/forgotPassword/index.jsx

```diff
@@ -0,0 +1,168 @@
+import React, { PureComponent } from 'react';
+import Modal from 'react-modal';
+import { FaTimes } from 'react-icons/fa';
+import { IoIosCheckmarkCircle } from 'react-icons/io';
+
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+import { forgotPasswordRequest } from '../../../services/auth';
+import validateAuth from '../../../validation/auth';
+import ShowMessage from '../showMessage';
+import Spinner from '../../../components/spinner';
```

Comment:
Fix import order: main react libraries- custom react libraries/default libraries - your components - styles. Separate each group by whitespace. Check all you files.